RDVFSharp/Commands/General/Status.cs

```diff
@@ -11,13 +12,55 @@ public class Status : BaseCommand<RDVFPlugin>
 
         public override async Task ExecuteCommand(string character, IEnumerable<string> args, string channel)
         {
-            if (Plugin.GetCurrentBattlefield(channel).IsInProgress)
+            if (channel == "ADH-a823a4e998a2b3d31794")
             {
-                Plugin.FChatClient.SendPrivateMessage(Plugin.GetCurrentBattlefield(channel).OutputController.LastMessageSent, character);
+                var argsList = args.ToList();
+                var NamedChannel = string.Join(" ", args);
+
+                if (NamedChannel == "Arena" || NamedChannel == "arena")
+                {
+                    Plugin.FChatClient.SendPrivateMessage(Plugin.GetCurrentBattlefield("ADH-b3c88050e9c580631c70").OutputController.LastMessageSent, character);
+                }
+
+                else if (NamedChannel == "Venue" || NamedChannel == "venue")
+                {
+                    Plugin.FChatClient.SendPrivateMessage(Plugin.GetCurrentBattlefield("ADH-51710b5ac8cce7e99f19").OutputController.LastMessageSent, character);
+                }
+                
+                else
+                {
+                    if (Plugin.GetCurrentBattlefield("ADH-b3c88050e9c580631c70").IsInProgress && Plugin.GetCurrentBattlefield("ADH-51710b5ac8cce7e99f19").IsInProgress)
```

Comment:
Same comment as before, use "public static string RDVF_VENUE_ROOMID" instead of inline strings like that